<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-MRVJBNT" failures="0" tests="8" name="com.menumitratCommonAPITestScript.CreateOrderTestScript" time="3.463" errors="3" timestamp="27 May 2025 15:59:39 GMT">
  <testcase classname="com.menumitratCommonAPITestScript.CreateOrderTestScript" name="createParcelOrderString" time="0.477"/>
  <testcase classname="com.menumitratCommonAPITestScript.CreateOrderTestScript" name="createCounterOrder" time="0.367"/>
  <testcase classname="com.menumitratCommonAPITestScript.CreateOrderTestScript" name="createOrderNegativeTest" time="0.193"/>
  <testcase classname="com.menumitratCommonAPITestScript.CreateOrderTestScript" name="createDeliveryOrder" time="0.574"/>
  <testcase classname="com.menumitratCommonAPITestScript.CreateOrderTestScript" name="createOrderNegativeTest" time="0.254">
    <error message="Error in create order negative test: A JSONObject text must begin with &#039;{&#039; at 1 [character 2 line 1]" type="com.menumitra.utilityclass.customException">
      <![CDATA[com.menumitra.utilityclass.customException: Error in create order negative test: A JSONObject text must begin with '{' at 1 [character 2 line 1]
	at com.menumitratCommonAPITestScript.CreateOrderTestScript.createOrderNegativeTest(CreateOrderTestScript.java:952)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- createOrderNegativeTest -->
  <testcase classname="com.menumitratCommonAPITestScript.CreateOrderTestScript" name="createOrderNegativeTest" time="0.203">
    <error message="Error in create order negative test: A JSONObject text must begin with &#039;{&#039; at 1 [character 2 line 1]" type="com.menumitra.utilityclass.customException">
      <![CDATA[com.menumitra.utilityclass.customException: Error in create order negative test: A JSONObject text must begin with '{' at 1 [character 2 line 1]
	at com.menumitratCommonAPITestScript.CreateOrderTestScript.createOrderNegativeTest(CreateOrderTestScript.java:952)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- createOrderNegativeTest -->
  <testcase classname="com.menumitratCommonAPITestScript.CreateOrderTestScript" name="createDriveThroughOrder" time="0.443"/>
  <testcase classname="com.menumitratCommonAPITestScript.CreateOrderTestScript" name="createDineInOrder" time="0.952">
    <error message="Exception in create Dine-In order test: Status code mismatch - Expected: 201, Actual: 400" type="com.menumitra.utilityclass.customException">
      <![CDATA[com.menumitra.utilityclass.customException: Exception in create Dine-In order test: Status code mismatch - Expected: 201, Actual: 400
	at com.menumitratCommonAPITestScript.CreateOrderTestScript.createDineInOrder(CreateOrderTestScript.java:367)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- createDineInOrder -->
</testsuite> <!-- com.menumitratCommonAPITestScript.CreateOrderTestScript -->
